demo-app/src/ClipPathPreview.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import styled from "styled-components";
+
+import Shape from "react-clip-path"
+
+const Playground = styled.div`
+  width: 100%;
+`;
+
+const ShapeDetails = styled.ul`
+  background-color: #ebebeb;
+  border-radius: 4px;
+  padding: 10px;
+  width: 100%;
+`;
+
+function ClipPathPreview(props) {
+  return(
+    <>
+      <Playground>
```

Comment:
Is this portion of the code copied from the App.js file? Can we please try creating component for the common code and use it in both the places instead?

Reply:
Done, moved all of the styled components to a separate file and both places import from it.

Reply:
Thanks! But still, we can create common components from this file. For example, this is a repeated code.

```js
<ShapeDetails>
          <li>
            <strong>Did you know?</strong> <p>{props.formInputs.notes}</p>
          </li>
          <li>
            <span>
              <b>Edges:</b> {props.formInputs.edges}
            </span>
          </li>
          <li>
            <span>
              <b>Vertices:</b> {props.formInputs.vertices}
            </span>
          </li>
          <li>
            <span>
              <b>clip-path:</b> <code><b>{props.formInputs.formula}</b></code>
            </span>
          </li>
        </ShapeDetails>
```

Reply:
Made this part of the code into its own component called ShapeDetailsBox